css/properties/display.json

```diff
@@ -51,30 +51,77 @@
             "deprecated": false
           }
         },
-        "list-item": {
+        "multi-keyword_values": {
           "__compat": {
-            "description": "<code>list-item</code>",
+            "description": "Multi-keyword values",
             "support": {
-              "webview_android": {
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": null
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              },
+              "samsunginternet_android": {
                 "version_added": null
               },
+              "webview_android": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
```

Comment:
Multi-keyword values are at risk, and may be dropped during the CR period, just like `run-in`, which has `"experimental": true`.